core/controllers/editor.py

```diff
@@ -953,11 +956,14 @@ def put(self, exploration_id):
         except utils.ValidationError as e:
             # We leave any pre-existing draft changes in the datastore.
             raise self.InvalidInputException(e)
-
+        exp_user_data = user_models.ExplorationUserDataModel.get(
+            self.user_id, exploration_id)
+        draft_change_list_id = exp_user_data.draft_change_list_id
         # If the value passed here is False, have the user discard the draft
```

Comment:
Can you reword this to say: "If is_version_of_draft_valid is False, ..."? The issue is that, with the addition of the new key to the dict, the referent of "the value passed here" is now ambiguous.